qt_gui_cpp/CMakeLists.txt

```diff
@@ -42,6 +42,9 @@ set(qt_gui_cpp_HDRS
   include/qt_gui_cpp/plugin.h
   include/qt_gui_cpp/plugin_bridge.h
   include/qt_gui_cpp/plugin_context.h
+  include/qt_gui_cpp/plugin.hpp
+  include/qt_gui_cpp/plugin_bridge.hpp
+  include/qt_gui_cpp/plugin_context.hpp
```

Comment:
I'm not entirely certain of this, but I don't believe we want to add these headers here.  The reason is that there are no SIGNAL/SLOT definitions in this file, so when the MOC generator runs on it, it will emit a warning about that.

That said, we should verify this.

Reply:
If I remove one of those, then `qt_gui_cpp_shiboken` test is failing becuase it's not able to find some symbols